src/emotiq.asd

```diff
@@ -37,7 +37,7 @@
                 :components ((:file "blockchain")))))
                                        
 (defsystem "emotiq/startup"
-  :depends-on (crypto-pairings)
+  :depends-on (crypto-pairings actors)
```

Comment:
@guitarvydas 

Adding this dependency isn't strictly related to removing watchdog timers, right?  

Rather this is necessary to build the system binary because of the use of `actors:install-actor-system`.   

Not going to hold up the commit for this reason, but would like you to comment so that my understanding is clear.

Reply:
Yes, I think you are correct.  "Actors" was added when testing the build on gitlab (which originally failed on binary builds).  Not strictly for removing the watchdog.